### ISSUE TITLE
Didnt save first time copied code in a blank contract now i cant compile to get back to running transaction to WITHDRAWL my eth *reward given if someone can help*

### ISSUE THREAD

**Issue by Dprice2413** [ethereum/remix-project#3075]
Guys Please help! to sum it up i copied the code first time created deployed and ran the contract i didnt save it so i tried to just do the same and something went wrong! i cant get to "at address" to run transactions on the original contract so i can pull my eth out of it. Please help! i have the transaction  for creation and deploying (runnning) I will give 150 in eth to anyone who helps me $200 if done before friday i just want to pull the contract like the first time so i can run transactions and withdrwal. Hope this helps

trans has: 0xe95d2a0e64105c9b64a27c981d6c9a66eae284b43ad97328fe5a40f37c09b58a 
link: https://etherscan.io/tx/0xe95d2a0e64105c9b64a27c981d6c9a66eae284b43ad97328fe5a40f37c09b58a
trans hash: 0xe95d2a0e64105c9b64a27c981d6c9a66eae284b43ad97328fe5a40f37c09b58a ( creation)
https://etherscan.io/tx/0xe95d2a0e64105c9b64a27c981d6c9a66eae284b43ad97328fe5a40f37c09b58a

please  help :(  its showing 0 deployed contracts for that address -   0x3f2492ddad8fb6197cc667b88f457d68d248d8b8 ( contract addy) but i know the api wasnt generated i just need to find out what to do so i can withdrawl my eth that i funded it with please :( 

@drafish  @Aniket-Engg  I know im annoyiong and a total noob i hope this help i will provide money for all your guys help im at that point
![Screenshot (139](https://user-images.githubusercontent.com/107596718/199568687-a95d21d5-2383-4ebf-942a-d6f3ef44067f.png)
![Screenshot (137)](https://user-images.githubusercontent.com/107596718/199568894-16d85903-7034-4da6-a1f8-e1101e8749af.png)
![Screenshot (138)](https://user-images.githubusercontent.com/107596718/199568905-d93767ff-20a5-4213-a70d-a866bc7dad74.png)
![Screenshot (139)](https://user-images.githubusercontent.com/107596718/199568908-939a4072-73b4-41c7-b2f5-4bc23fefcfa7.png)

**Comment by drafish** [comment]
I think you may run into a scam. I meet two guys who was scamed too on remix's gitter chat. They saw a youtube video that can teach them how to make $1000/DAY on UniSwap. So they followed this video step by step, deployed the contract, sent eth to the contract, then clicked the start button. Finally their eth was transfered to the scammer's address. 

Remix team warned this kind of scams on the home page. They even wrote an article about that. Check this. https://medium.com/remix-ide/remix-in-youtube-crypto-scams-71c338da32d

Did you aware that you were using two version of remix. `https://remix.ethereum.org/` is the official link. `https://remix-eth.github.io/` is provide by scammer and the scam alert was removed by scammer on purpose.

Just remember always use the official link. But in your case, it's already too late. Once you send eth to the contract's address, there is no way to get it back.

**Comment by Dprice2413** [comment]
@drafish  Soo nooo thats the weird thing bro, i know theres front runner bots for scams but the contract was made by me and i deployed and ran it. My eth hasnt moved to any address its still there,. I created that contract and funded it, and ran it. Thats the weird part if it was a scam ive seen multiple examples where it transfers to the scammers wallet but this isnt  the case. The code clearly states ( send eth directly to the wallet of the person who deployed the contract and my address since i used injected web3 (metamask) my address was embedded in the code. Sooo you or someone said that they were able to "run transactions and find the contract with the "@ address. wasnt it you? You told me you were able to get to that point. The network congestion isnt MY issue. I didnt save the orginal contract i deployed my comp had an update and im dumb cleared the cache. It was up and i was able to run trasnactions and deploy and start. The weird thing is i copied the same code and tried to compile to the code to run transactions again but for some reason copying and pasting doesnt copy the same. Sooo i hope you understand. The eht is there it hasnt been touched or transfered since i deployed and ran it. All i need is to Compile the code and enter the contract address soo i can run the trasnaction of "withdrawl" didnt you tell me you coded it where it worked ? I need to get to THAT POINT. i have the page with the copy and pasted code. If you can get me to this point on the orginal remix-ehterium soo i can pull the eth i will give you $200 immediately as long before saturday. Like i believe it was you that did it. Soo please please if we can get to that point to where i can compile the data & run the withdrawl transaction and search by the contract address, @drafish  im 100% sure it was you. Let me give you the link of the Code i copied and pasted maybe you can point out the error but i know the scam going on but this is not the case the funds and eth are still in my contract and the coding  code states ( back to wallet address of contract creator something like that don't quote me on the exact formula but here is the link to the site that i copied and pasted the code. Im sorry i really dont mean to bother you but i will def pay you for you time and effort im reallty sorry and im mosst defitley never touching code ever again >.< i will pay you thanks again.


Also dont know if this helps but this was in the code. My buddy did it i know theres those scams going around but this wasnt it. This was from a private group he was able to it but i need to get to the point where i can run trasnactions
this was embedded in the code 

" 
     * @dev withdrawals profit back to contract creator address
     * @return `profits`.
     */
    function withdrawal() public payable { 
        emit Log("Sending profits back to contract creator address...");
        payable(withdrawalProfits()).transfer(address(this).balance);"

**Comment by drafish** [comment]
Hey buddy. I take a look at your contract's transactions. It does look weird. I saw the transaction that you called the start method. It should transfer all the contract's balance away. But instead of that, it only transfer 1 wei to another address. 
![image](https://user-images.githubusercontent.com/18543527/201007405-e3057cf0-33f8-47d3-bb5f-a2d93cb1cdc4.png)
Is this your address? Did you change the scammer's contract code so that the contract will send ether back to your address?

Anyway, I think I know why you can't compile your contract. It because you got the suffix of the contract's filename wrong. It shoud be `.sol` not `.eth`. Rename your filename and try again.

No need for money. Happy to help. @Dprice2413

**Comment by Dprice2413** [comment]
Nooo nooo I get that, but look when I ran when contract the eth stayed on
the contract that I created and deployed. The eth is still there! I
actually swapped addresses to mine when compiling first code. You stated
that when the two others ran it they lost their eth went to the scammers
wallet mine hasn't. I still have it, can you just please help me get to the
point where I can withdrawl. I'll give you $100 even if when I run the
transaction it goes poof. Please I just need to get to that point that's
all. The eth hasn't moved since I created everything. If I can run the
withdrawl transaction myself I'll pay you. Even if it takes my $ or eth
I'll give you $100 if it works I'll give you $200 please that's all I
need.  Thank you

**Comment by drafish** [comment]
Yeah, I understand what you say. Maybe I should give you the detail steps.

1. Rename your contract's filename with correct suffix which is `.sol`
2. Compile your contract. 
3. Load your contract by the `AtAddress` button.
4. Run the withdrawal transaction.

But I'm not sure this will actually withdraw your ether. I hope you know the contract you deployed.

**Comment by Dprice2413** [comment]
Sooo you think it didn't compile because of the name ? If I rename it would
that fix it ?
My main issue is I literally copied and pasted the same code I used
originally and it wouldn't compile that's why I'm confused. I also do have
the contract address and transaction hash. I just wanna try thank you if
this works I'll pay u. Question if you read the contract I deployed would u
be able to distinguish what's the issue ?! Or idk it's weird if I copied
and pasted the same exact code it should compile. I literally know the
steps that not the issue the problem is I copied and pasted like I did the
first time and it won't compile and thags why I'm confused I entered the
address of the contract at address and because it wouldn't compile it
wouldn't let me and greyed out
